lib/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -140,14 +140,21 @@ public String getSigningKeyId() {
         return ssmProvider.get(getEnvironmentVariable(SIGNING_KEY_ID_PARAM));
     }
 
-    public CredentialIssuerConfig getCredentialIssuer(String credentialIssuerId) {
+    public CredentialIssuerConfig getCredentialIssuerActiveConnectionConfig(
+            String credentialIssuerId) {
         Map<String, String> result =
                 getSsmParameters(
                         String.format(
-                                "%s/%s",
+                                "%s/%s/connections/%s",
```

Comment:
I thought, we can take connections static path from different params. So we can manage configuration changes with params and without code changes.